spec/services/dttp/contact_update_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Dttp
+  describe ContactUpdate do
+    describe "#call" do
+      it_behaves_like "dttp request"
```

Comment:
I don't think we should split this out here. The shared example is quite specific to this operation. There are definitely bits of it that we could reuse but best to do that if we actually reuse them. I'm also not too fussed on being overly DRY in the specs as they're a pain to follow when they're spread over multiple files.

Reply:
Refactored, now it's contained within its own test